app/(technician)/layout.tsx

```diff
@@ -13,15 +13,17 @@ export default function FarmerLayout({
   if (role === "FARMER") redirect("/dashboard")
```

Comment:
Should be a constant

```suggestion
  if (role === FARMER_ROLE) redirect("/dashboard")
```